pkg/cmd/pulumi/login.go

```diff
@@ -171,6 +172,7 @@ func newLoginCmd() *cobra.Command {
 	cmd.PersistentFlags().StringVar(&defaultOrg, "default-org", "", "A default org to associate with the login. "+
 		"Please note, currently, only the managed and self-hosted backends support organizations")
 	cmd.PersistentFlags().BoolVarP(&localMode, "local", "l", false, "Use Pulumi in local-only mode")
+	cmd.PersistentFlags().BoolVarP(&localMode, "insecure", "k", false, "Allow insecure server connections when using SSL")
```

Comment:
Also, why `-k` for the shorthand char? Do we even need a shorthand for this one?